tools/cli/adminCommands.go

```diff
@@ -205,7 +205,7 @@ func AdminListNamespaces(c *cli.Context) {
 		}
 		return items, token, nil
 	}
-	paginate(c, paginationFunc)
+	paginate(c, paginationFunc, []string{})
```

Comment:
Use `nil` instead of empty slice. Everywhere. Always. All go funcs such as `append` or `len` work perfectly with `nil`.